suites/pacific/cephfs/tier_2_fs.yaml

```diff
@@ -112,3 +112,34 @@ tests:
       polarion-id: CEPH-83574592
       desc: Concurrent-clone-test
       abort-on-fail: false
+  - test:
+      name: Clone_status
+      module: test_clone_status.py
+      polarion-id: CEPH-83573501,CEPH-83573504,CEPH-83573502
+      desc: Checks the clone status and states of the clone process
+      abort-on-fail: false
+  - test:
+      name: Retain_Snapshots
+      module: test_retain_snapshots.py
+      polarion-id: CEPH-83573521,CEPH-83573415
+      desc: Retains the snapshots after deletig the subvolume
+      abort-on-fail: false
+  - test:
+        name: Remove_Subvolume_clone
+        module: test_clone_remove_subvol.py
+        polarion-id: CEPH-83573499
+        desc: Clone a subvolume and remove the orginal volume and verify the contents in subvolume
+        abort-on-fail: false
+  - test:
+      name: Test Max Snapshot limit
+      module: test_max_snapshot_limit.py
+      polarion-id: CEPH-83573520
+      desc: Validate the max snapshot that can be created under a root FS sub volume level.Increase by 50 at a time until it reaches the max limit.
+      abort-on-fail: false
+  - test:
+      name: Snapshot reboot
+      module: test_snapshot_reboot.py
+      polarion-id: CEPH-83573418,CEPH-83573420
+      desc: Create a Snapshot, reboot the node and rollback the snapshot
+      abort-on-fail: false
+
```

Comment:
Would it make sense having a separate suite?

Reply:
@AmarnatReddy we are having multiple polarion tests mapped to one test script?
Can we have 1-to-1 mapping of Polarion tests to automated test script?
 CEPH-83573501,CEPH-83573504,CEPH-83573502

Reply:
If needed merge the Polarion tests.

Reply:
Hi Veera,
I am segregating the test scripts to have 1-to-1 mapping.